torch/fx/passes/fake_tensor_prop.py

```diff
@@ -29,10 +31,29 @@ def __init__(self, module: torch.fx.GraphModule, mode: Optional[FakeTensorMode]
 
     def run_node(self, n: Node):
         result = super().run_node(n)
-        n.meta['val'] = result
+
+        def extract_val(obj):
+            if isinstance(obj, FakeTensor):
+                return snapshot_fake(obj)
+            elif isinstance(obj, torch.Tensor):
+                return snapshot_fake(self._mode.from_tensor(obj))
+            elif isinstance(obj, py_sym_types):
+                return obj
+            else:
+                return None
```

Comment:
I am experimenting that using symbolic mode of fx.graph with FakeTensorProp to get shapes, and further use it to validate torch/onnx op level results inside a dynamic model. Before this PR, the obj (int) could be propagated to fx.node (ex: sym_size). Is here intentionally blocking the usage? If so, could you suggest an idiomatic way to do it?